src/main/java/seedu/partyplanet/logic/commands/EAddCommand.java

```diff
@@ -19,7 +20,7 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an event to PartyPlanet. "
             + "Parameters: "
             + PREFIX_NAME + " NAME "
-            + "[" + PREFIX_BIRTHDAY + " BIRTHDAY CELEBRATION DATE] "
+            + "[" + PREFIX_DATE + " EVENT DATE] "
```

Comment:
Here can just leave it as `DATE`.
Since to the user, event date is basically a date (unlike birthday where they are extra restriction).
Or if decide to keep, put it as one word `EVENTDATE`, or else it seems like 2 arguments

```suggestion
            + "[" + PREFIX_DATE + " DATE] "
```